docs/en/ingest-management/elastic-agent/install-fleet-managed-agent.asciidoc

```diff
@@ -0,0 +1,29 @@
+["appendix",role="exclude",id="install-fleet-managed-agent"]
+= Install a {fleet} managed {agent}
+
+This guide describes how to:
+
+* Install an {agent} that will be managed with {fleet}
+* Enroll the {agent} with {fleet}
+
+If you have come here from the {agent} download page, then you are in the right place.
+
+[discrete]
+[[minimal-add-agent-to-fleet]]
+== Step 1: Add an {agent} to {fleet}
+
+{agent} is a single, unified agent that you can deploy to hosts or containers to
+collect data and send it to the {stack}. Behind the scenes, {agent} runs the
+{beats} shippers or {elastic-endpoint} required for your configuration.
+
+Choose the operating system and run the provided commands to download and extract
+{agent} on your system.
+
+include::{fleet-repo-dir}/elastic-agent/install-fleet-managed-elastic-agent.asciidoc[tag=install-elastic-agent]
+
+[discrete]
+[[enroll-agent]]
+== Step 2: Enroll and start the {agent}
+
+Return to {kib} and follow the **Enroll and start the Elastic Agent** instructions.  The command provided in {kib} includes the information that {agent} needs to connect and authenticate with the {stack}.  When the {agent} enrolls it will be configured to send the data associated with the policy that you created.
```

Comment:
What about users who go to the download page before they look at Kibana? We should provide something that makes sense for them. Maybe it's enough to mention the path to the Add Agent flyout?